src/som/Output.java

```diff
@@ -0,0 +1,82 @@
+package som;
+
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+
+import som.vm.VmSettings;
+
+
+public class Output {
+
+  public static class AnsiColor8 {
+    public static final String BLACK   = "\u001b[30m";
+    public static final String RED     = "\u001b[31m";
+    public static final String GREEN   = "\u001b[32m";
+    public static final String YELLOW  = "\u001b[33m";
+    public static final String BLUE    = "\u001b[34m";
+    public static final String MAGENTA = "\u001b[35m";
+    public static final String CYAN    = "\u001b[36m";
+    public static final String WHITE   = "\u001b[37m";
+    public static final String RESET   = "\u001b[0m";
+  }
+
+  @TruffleBoundary
+  public static void print(final String msg, final String color) {
+    // Checkstyle: stop
+    if (VmSettings.NO_ANSI_COLOR_IN_OUTPUT) {
+      System.out.print(msg);
```

Comment:
This isn't a correct patch.
We used to output to `System.err` when showing errors.